documentation/modules/configuring/ref-storage-persistent.adoc

```diff
@@ -5,36 +5,48 @@
 [id='ref-persistent-storage-{context}']
 = Persistent storage
 
-Persistent storage uses {K8sPersistentVolumeClaims} to provision persistent volumes for storing data.
-Persistent Volume Claims can be used to provision volumes of many different types, depending on the {K8SStorageClass} which will provision the volume.
-The data types which can be used with persistent volume claims include many types of SAN storage as well as {K8sLocalPersistentVolumes}.
+[role="_abstract"]
+Persistent data storage retains data even when a pod that uses the storage restarts.
+Persistent data storage uses a storage framework called persistent volumes (PV). 
```

Comment:
MAybe we should call it _Dynamic Volume Provisioning_? https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/

Reply:
I think we can use that term
I've rewritten and reordered these lines a little

> A dynamic provisioning framework enables clusters to be created with persistent storage. 
> Pod configuration uses {K8sPersistentVolumeClaims} (PVCs) to make storage requests on persistent volumes (PVs).
> PVs are storage resources that represent a storage volume.
> PVs are independent of the pods that use them.